test/james.testing.clss.pas

```diff
@@ -0,0 +1,53 @@
+unit james.testing.clss;
+
+{$include james.inc}
+
+interface
+
+uses
+    james.testing,
+  {$IFDEF FPC}
+    fpcunit, james.testing.clss.lazarus
+  {$ELSE}
+    TestFramework, james.testing.clss.delphi
+  {$ENDIF}
+  ;
+
+type
+  {$IFDEF FPC}
+    TTestCase = FPCUnit.TTestCase;
+  {$ELSE}
+    TTestCase = TestFramework.TTestCase;
+  {$ENDIF}
```

Comment:
@nunopicado here it should have and *alias* to `TTest` like this:
```pascal
  {$IFDEF FPC}
    TTestCase = FPCUnit.TTestCase;
    TTest = James.Testing.FPC.TTest<TTestCase>;
  ...
```
This syntax compiles (at least on FPC) at this part of the code, but I got an error in another:
`james.data.tests.pas(111,31) Error: Identifier not found "TTest$1$crc5764DDEA$1"`

I don't know if the syntax isn't correct or if FPC does not implement "alias for generics".